packages/node/src/indexer/indexer.manager.ts

```diff
@@ -481,111 +448,105 @@ export class IndexerManager {
     }
   }
 
-  private getDataSourcesForSpecName(): SubqlProjectDs[] {
-    return this.project.dataSources.filter(
-      (ds) =>
-        !ds.filter?.specName ||
-        ds.filter.specName === this.api.runtimeVersion.specName.toString(),
-    );
-  }
-
   private async indexBlockData(
-    { block, events, extrinsics }: BlockContent,
+    { block, events, messages, transactions }: BlockContent,
     dataSources: SubqlProjectDs[],
     getVM: (d: SubqlProjectDs) => IndexerSandbox,
   ): Promise<void> {
     await this.indexBlockContent(block, dataSources, getVM);
 
-    // Run initialization events
-    const initEvents = events.filter((evt) => evt.phase.isInitialization);
-    for (const event of initEvents) {
-      await this.indexEvent(event, dataSources, getVM);
+    for (const tx of transactions) {
```

Comment:
Nice work getting the messages in here.

We want this to behave differently though, we now index things in order of time.

So something like this:

```
transaction0
  msg0
    event0
  msg1
transaction1
  msg0
    event0
    event1
```
So for each transaction, we process it, then each message and for each message we process the event.